src/js/program/media-element-pool.js

```diff
@@ -64,6 +65,12 @@ function primeMediaElementForPlayback(mediaElement) {
     // If we're in a user-gesture event call load() on video to allow async playback
     if (!mediaElement.src) {
         mediaElement.load();
+    } else if (OS.android && !mediaElement.parentNode) {
+        // If the player sets up without a gesture and preloads, the background tag may not be primed for playback.
+        // We need to load again on Android in order to play without another gesture
+        if (!mediaElement.played.length) {
```

Comment:
Combine this if with the else if:
`else if (OS.android && !mediaElement.parentNode && !mediaElement.played.length)`